.circleci/config.yml

```
@@ -202,3 +217,6 @@ workflows:
       - test-go:
           requires:
             - build-go-dev
+      - test-ember:
+          requires:
```

This job is going to require `install-ui-dependencies` too because it restores the Yarn cache.
```suggestion
          requires:
            - install-ui-dependencies
```